src/harness/testcases/README.md

```diff
@@ -4,3 +4,39 @@
 
 The current test cases are based on the [WinnForum test specification](https://workspace.winnforum.org/higherlogic/ws/public/download/5637/WINNF-TS-0061-V1.1.0%20-%20WG4%20SAS%20Test%20and%20Certification%20Spec.pdf) v1.1 (Jan 2018)
 
+## Security Test Cases
+
+The SCS.11/16, SDS.11/16 and SSS.11/16 TCs just expect the handshake failure and do not validate the reason for failure.<br>
+The correct way to run the fake_sas for these test cases is
+<pre>
+<code>
+<b> $ python fake_sas.py --verify_crl --ca ca_crl_chain_sxs11.cert </b>
+</code>
+</pre>
+
+To obtain the CRL file, the <b>simple CRL server</b> in <b>master</b> branch needs to be 
```

Comment:
You've listed the easy way first, which is good, but now the remaining instructions are inconsistent, because `ca_crl_chain_sxs11.cert` was provided automatically.

Either clarify the circumstances under which someone would manually generate a CRL file, or delete this section.

Reply:
We have updated  README.md section to reflect the WG.2 agreements that were concluded to use CRL Server and the agreed test procedures.